docs/index.md

```diff
@@ -40,7 +40,20 @@ addSbtPlugin("org.typelevel" % "sbt-typelevel-ci-release" % "@VERSION@")
 
 @:@
 
-Then configure your build.
+
+### Giter8 Template
+
```

Comment:
I think we should move this section higher up :) since the `plugins.sbt` stuff isn't necessary for someone using the g8, nor is the choice between sbt-typelevel and sbt-typelevel-ci-release.

Reply:
Good call, it's awkward in the middle here, but I wasn't sure if we wanted it front and center.